server_admin/topics/sso-protocols/docker.adoc

```diff
@@ -17,6 +17,10 @@ NOTE: This flow assumes that a `docker login` command has already been performed
  * The Docker client will get the bearer token from the JSON response and use it in the Authorization header to request the protected resource.
  * When the Docker registry receives the new request for the protected resource with the token from the {project_name} server, the registry validates the token and grants access to the requested resource (if appropriate).
 
+NOTE: There is no user session created on the {project_name} side after successful authentication with the Docker protocol. Docker protocol is
+not used in case of browser SSO session and it does not have a way to refresh token or ask {project_name} server if particular token/session
+is still valid. So creating the session would be unnecessary overhead for this protocol. More details in the <<_transient-session, transient session>> section.
```

Comment:
change "would be" to "is"
Change "More details in the"
to "For more details, see the"